src/components/cryptolist.module.css

```diff
@@ -0,0 +1,122 @@
+.cryptolist {
+  background: #0b0e1c;
+  color: white;
+}
+
+.unordered {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 20px;
+}
+
+.listItem {
+  width: 650px;
+  height: 320px;
+  padding-right: 0;
+  list-style-type: none;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 15px;
+}
+
+.listItem:nth-child(even) {
+  background: linear-gradient(90deg, #2d3874 0, #2e4d75);
+}
+
+.listItem:nth-child(odd) {
```

Comment:
- [ ] Since we are building a *mobile web application*, The UI doesn't look fit. Please style so that users can easily use this site in the mobile view.

### Mobile view issue

![Screenshot from 2022-08-11 01-05-39](https://user-images.githubusercontent.com/26767383/184006855-5febcbbf-34a5-493a-9cf2-97ddc923a4cc.png)